src/Microsoft.DotNet.Interactive.Http/Model/HttpNamedRequest.cs

```diff
@@ -171,19 +180,24 @@ internal HttpNamedRequest(HttpRequestNode httpRequestNode, HttpResponse response
                     var xmlDoc = new XmlDocument();
                     xmlDoc.LoadXml(Response.Content.Raw);
 
-
-                    //Remove the leading slash
-                    var xmlNodes = xmlDoc.SelectNodes(path[3].Substring(1));
-
-                    if (xmlNodes is not null && xmlNodes.Count == 1)
+                    try
                     {
-                        return node.CreateBindingSuccess(xmlNodes.Item(0)?.Value);
+                        //Remove the leading slash
+                        var xmlNodes = xmlDoc.SelectNodes(path[3].Substring(1));
+
+                        if (xmlNodes is not null && xmlNodes.Count == 1)
+                        {
+                            return node.CreateBindingSuccess(xmlNodes.Item(0)?.Value);
+                        }
+                        else
+                        {
+                            return node.CreateBindingFailure(HttpDiagnostics.InvalidXmlNodeInNamedRequest(path[3]));
+                        }
                     }
-                    else
+                    catch
```

Comment:
Do you have an example of the kinds of exceptions that will be caught here?

It's always preferable to catch specific exceptions to avoid hiding bugs.